git-hooks/commit/errors.js

```diff
@@ -0,0 +1,86 @@
+const TYPES = require('./types')
+const { format } = require('../util/formatting')
+
+/* STYLES */
+const h1 = format({ fg: 'red', bold: true })
+const h2 = format({ fg: 'cyan', bold: true })
+const code = format({ fg: 'gray' })
+
+
+/* SNIPPETS  */
+const SYNTAX = '<type>[(<scope>)][!]: <summary>'
+const SYNTAX_NO_SCOPE = '<type>[!]: <summary>'
+const TYPES_INLINE = Object.keys(TYPES).join(', ')
+
+
+/* ERROR FUNCTIONS */
+
+const incorrectFormat = (txtMsg) => console.error(`
+${h1('ERROR: Submitted commit message header does not match expected format!')}
```

Comment:
Indentation is intentional for ensuring consistent CLI output across all error messages (specifically `msgContent` in `unusableContent()`)